### ISSUE TITLE
Schema Compare: Generate Script Ignores checkboxes

### ISSUE THREAD

**Issue by Vayth** [microsoft/azuredatastudio#10723]
- Azure Data Studio Version: 1.18.1
- Windows 10

Steps to Reproduce:

1. Schema Compare 2 DB (tested from local source to azure dest & local source to azure dest)
2. Uncheck the checkboxes
3. Generate Scripts

The generated script still included the unchecked table. Not sure what to do since I didn't find other people having this problem online and this is kind of basic feature.

![image](https://user-images.githubusercontent.com/41632555/83721621-a64a0d00-a665-11ea-8764-d6b21339f5d0.png)

Generated Script:
![image](https://user-images.githubusercontent.com/41632555/83721656-b7931980-a665-11ea-8c97-5f95adaa678a.png)

Note: Visual Studio Schema Compare generates desired result.

**Comment by kisantia** [comment]
@Vayth does this repro every time and for other object types too? If generate script was clicked right away after unchecking the checkbox, it's possible that the exclude operation hadn't finished when the generate script started.

I was not able to repro this and tables changes were not included in the script as expected.

**Comment by Vayth** [comment]
After trying it again, it doesn't seem to happen every time. Still unsure what triggered it back then.

![image](https://user-images.githubusercontent.com/41632555/83941573-b1906a80-a816-11ea-91ea-dc0f0239bd3f.png)
The checkboxes stayed in their state, and what has been unchecked, still generated on the script.
However if I checked a table then unchecked it again, today, it doesn't get generated.

Any idea or other info I can give to help finding this intermittent behavior?

**Comment by kisantia** [comment]
@Vayth it could be related to #5939. For this issue, the unchecked checkboxes that weren't clicked to get to that state aren't actually excluded. For this, the workaround is to scroll down the whole table before unchecking anything. Did you manually uncheck all those checkboxes initially?

**Comment by Vayth** [comment]
Yes, that might be the case. I did uncheck all those checkboxes manually, and the DB did have a lot of table, although less than 20,000 like mentioned in the other issue. The problem didn't occur in my attempt to reproduce this issue, where I just used 3-4 tables.

On a side note, is there a way to uncheck multiple checkboxes at once? Unlike Schema Compare tool in Visual Studio, I couldn't select many, right click, and then select exclude.
